install/kubernetes/helm/istio/values.yaml

```diff
@@ -401,8 +401,9 @@ grafana:
   storageClassName: ""
   security:
     enabled: false
-    adminUser: admin
-    adminPassword: admin
+    secretName: grafana
+    usernnameKey: username
+    passworkdKey: password
```

Comment:
There are multiple typos here. Have you tested and verified this actually works?

Reply:
Done, thanks @douglas-reid